packages/blockchain/src/types.ts

```diff
@@ -82,6 +84,14 @@ export interface BlockchainOptions {
    */
   validateConsensus?: boolean
 
+  /**
+   * Options to pass in string to define consensus
+   * Or to pass in custom consensusAlgorithm that correctly implements `Consensus`
+   */
+
+  consensus?: 'casper' | 'clique' | 'ethash'
+  consensusAlgorithm?: (blockchain: { blockchain: Blockchain }) => Consensus
```

Comment:
What does this do?  I see the comment but I'm not following what it actually does since it doesn't appear to attach the consensus class to the `blockchain` object in the constructor.